diff --git a/core/makefile b/core/makefile
index c777aea7..ef9c4c8a 100644
--- a/core/makefile
+++ b/core/makefile
@@ -329,7 +329,7 @@ config : .null
 	-@$(SED) "s/\[major\]/$(PRO_VERSION_MAJOR)/g" $(PRO_DIR)/$(PRO_NAME).config.h
 	-@$(SED) "s/\[minor\]/$(PRO_VERSION_MINOR)/g" $(PRO_DIR)/$(PRO_NAME).config.h
 	-@$(SED) "s/\[alter\]/$(PRO_VERSION_ALTER)/g" $(PRO_DIR)/$(PRO_NAME).config.h
-	-@$(SED) "s/\[build\]/`date +%Y%m%d%H%M`/g" $(PRO_DIR)/$(PRO_NAME).config.h
+	-@$(SED) "s/\[build\]/197001010000/g" $(PRO_DIR)/$(PRO_NAME).config.h
 	-@$(SED) "s/\[debug\]/\($(if $(findstring y,$(DEBUG)),1,0)\)/g" $(PRO_DIR)/$(PRO_NAME).config.h
 	-@$(SED) "s/\[small\]/\($(if $(findstring y,$(SMALL)),1,0)\)/g" $(PRO_DIR)/$(PRO_NAME).config.h
 	-@$(SED) "s/\/\/.*\[packages\]/$(foreach name, $(PKG_NAMES), $(call MAKE_PACKAGE_INFO_,$($(name)_upper)))/g" $(PRO_DIR)/$(PRO_NAME).config.h
diff --git a/core/xmake.lua b/core/xmake.lua
index 590e1f48..c02ad7dc 100644
--- a/core/xmake.lua
+++ b/core/xmake.lua
@@ -2,7 +2,7 @@
 set_project("xmake")
 
 -- version
-set_version("2.3.2", {build = "%Y%m%d%H%M"})
+set_version("2.3.2", {build = "197001010000"})
 
 -- set xmake min version
 set_xmakever("2.2.3")
